### PR TITLE
Credere Staging Server

### PR DESCRIPTION
**Pull request by ghost**
Addressing issue #421

### CODE REVIEW

**Comment by RobHooper** [comment]
@jpmckinney I made a few assumptions in this PR that need reviewing before it is merged, such as:
* The URL / domain name the site will use
* Installing Docker Compose and adding a placeholder configuration file
* I have also enabled Basicauth

You can test the new server stack by setting the following in your hosts file:
> 178.79.139.218 credere.open-contracting.org

File: pillar/credere.sls [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  credere:
+    target: credere
+
+postgres:
+  version: 15
+  # Docker containers don't use localhost to connect to the host's PostgreSQL service. Public access is controlled using Linode's firewall.
+  public_access: True
+  configuration: False
+  databases:
+    credere_db:
```
Comment (jpmckinney) [inline pillar/credere.sls:33]:
```suggestion
    credere:
```

**Comment by jpmckinney** [comment]
> * The URL / domain name the site will use

Correct!

> * Installing Docker Compose and adding a placeholder configuration file

Looks good.

> * I have also enabled Basicauth

Sounds good. I assume the change is unstaged for pillar-private - feel free to commit directly.

**Comment by RobHooper** [comment]
Private pillar pushed. :+1:

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**